src/models.py

```diff
@@ -931,6 +959,45 @@ def _lm_forward(self, batch, task, predict):
             pass
         return out
 
+    def _lm_only_lr_forward(self, batch, task, predict):
+        """Only left to right pass for LM model - non-bidirectional models.
+        Args:
+            batch: indexed input data
+            task: (Task obejct)
+            predict: (boolean) predict mode (not supported)
```

Comment:
Explain what happens if you set this. If it's something subtly wrong, then you need an assert to prevent it from being set.

Reply:
Removed the predict flag, was redundant.